### PR TITLE
feat(facade): add facade for freeze and getCellData

### PR DESCRIPTION
**Pull request by weird94**
close #xxx

## Pull Request Checklist

- [ ] Related tickets or issues have been linked in the PR description (or missing issue).
- [ ] [Naming convention](https://github.com/dream-num/univer/blob/dev/docs/NAMING_CONVENTION.md) is followed (**do please** check it especially when you created new plugins, commands and resources).
- [ ] Unit tests have been added for the changes (if applicable).
- [ ] Breaking changes have been documented (or no breaking changes introduced in this PR).

### CODE REVIEW

File: packages/facade/src/apis/sheets/f-range.ts [hunk 1]
```diff
@@ -255,6 +255,24 @@ export class FRange {
         return range;
     }
 
+    /**
+     * Returns the cell data for the cells in the range.
+     * @returns A two-dimensional array of cell data.
+     */
+    getCellDatas(): Nullable<ICellData>[][] {
```
Comment (hexf00) [inline packages/facade/src/apis/sheets/f-range.ts:262]: `Datas` is not a canonical plural form, consider `getCellDataGrid` or `getCellDataTable`.

File: packages/facade/src/apis/sheets/f-worksheet.ts [hunk 2]
```diff
@@ -947,4 +948,28 @@ export class FWorksheet {
      * @param range The range to set as the active selection.
      */
     setActiveSelection = this.setActiveRange;
+
+    /**
+     * Sets the frozen state of the current sheet.
+     * @param {IFreeze} freeze - The freeze object containing the parameters for freezing the sheet.
+     * @returns {boolean} True if the command was successful, false otherwise.
+     */
+    setFreeze(freeze: IFreeze): boolean {
```
Comment (hexf00) [inline packages/facade/src/apis/sheets/f-worksheet.ts:957]: Can the following API be implemented?

- [setFrozenColumns(columns)](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet#setFrozenColumns(Integer))	void	Freezes the given number of columns.
- [setFrozenRows(rows)](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet#setFrozenRows(Integer))	void	Freezes the given number of rows.
- [getFrozenColumns()](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet#getFrozenColumns())	Integer	Returns the number of frozen columns.
- [getFrozenRows()](https://developers.google.com/apps-script/reference/spreadsheet/spreadsheet#getFrozenRows())	Integer	Returns the number of frozen rows.

**Review by hexf00 — CHANGES_REQUESTED** [review]
Some suggested changes

File: packages/facade/src/apis/sheets/f-worksheet.ts (hunk 2 above)
Comment (hexf00) [reply packages/facade/src/apis/sheets/f-worksheet.ts:957]: and hope you can add some test code

Comment (weird94) [reply packages/facade/src/apis/sheets/f-worksheet.ts:957]: add getFreeze

**Review by hexf00 — APPROVED** [review]

**Comment by hexf00** [comment]
I'll add
setFrozenColumns(columns) void Freezes the given number of columns.
setFrozenRows(rows) void Freezes the given number of rows.
getFrozenColumns() Integer Returns the number of frozen columns.
getFrozenRows() Integer Returns the number of frozen rows.

**Pull request merged**